utils/seed.js

```diff
@@ -0,0 +1,112 @@
+const axios = require("axios")
+const { PrismaClient } = require("@prisma/client")
+const client = new PrismaClient()
+
+const fetchRandomUsers = async () => {
+    try {
+        const response = await axios.get(
+            "https://api.freeapi.app/api/v1/public/randomusers?page=1&limit=10",
+        )
+        return response.data.data
+    } catch (error) {
+        console.error("Error fetching random users:", error.message)
+        throw error
+    }
+}
+
+const fetchRandomProducts = async () => {
+    try {
+        const response = await axios.get(
+            "https://api.freeapi.app/api/v1/public/randomproducts?page=1&limit=10&inc=title%2Cdescription%2Ccategory%2Cstock%2Cprice%2Cimages&query=fashion",
+        )
+        return response.data.data
+    } catch (error) {
+        console.error("Error fetching random products:", error.message)
+        throw error
+    }
+}
+
+const seed = async () => {
+    try {
+        const usersResponse = await fetchRandomUsers()
+        const productsResponse = await fetchRandomProducts()
+
+        const users = usersResponse.data
+        const products = productsResponse.data
+
+        // Insert users
+        for (const user of users) {
+            await client.user.create({
+                data: {
+                    name: `${user.name.first} ${user.name.last}`,
+                    email: user.email,
+                    password: user.login.password,
+                },
+            })
+        }
+
+        // Fetch all user IDs from the database
+        const allUsers = await client.user.findMany({
+            select: {
+                id: true,
+            },
+        })
+
+        const userIds = allUsers.map((user) => user.id)
```

Comment:
Remove it, It is not necessary.
you can simply use `const randomUserId = allUsers[Math.floor(Math.random() * allUsers.length)].id;`